src/auth/strategies/jwt.strategy.ts

```diff
@@ -0,0 +1,33 @@
+import { Injectable, UnauthorizedException } from '@nestjs/common';
+import { ConfigService } from '@nestjs/config';
+import { PassportStrategy } from '@nestjs/passport';
+import { ExtractJwt, Strategy } from 'passport-jwt';
+
+import { UserRepository } from '@src/user/repositories/user.repository';
+import { JwtPayload } from '../types/jwt.interface';
+
+enum MESSAGES {
```

Comment:
This could be moved to the constant i mentioned before

Reply:
done